github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1881,6 +1881,35 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
+    annotations:
+      fork-per-release: "true"
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    max_concurrency: 11
+    name: pull-kubevirt-code-lint
+    optional: true
+    skip_branches:
+    - release-\d+\.\d+
+    skip_report: true
```

Comment:
why `skip_report` true ? 
we do want to see the result on the PR, `optional` true is enough

btw imho we should not add fields that have the default value, this is the common practice in project-infra
but this is other story

Reply:
> why `skip_report` true ? we do want to see the result on the PR, `optional` true is enough

I agree.

> btw imho we should not add fields that have the default value, this is the common practice in project-infra but this is other story, it is confusing and potentially leading to mistakes (i.e copy paste)

I disagree
You are requiring me to remember the defaults instead of seeing the settings ecplicitly.

Reply:
> > btw imho we should not add fields that have the default value, this is the common practice in project-infra but this is other story, it is confusing and potentially leading to mistakes (i.e copy paste)
> 
> I disagree You are requiring me to remember the defaults instead of seeing the settings ecplicitly.

Imo, it isn't a sufficient reason for dragging those fields with us again and again.
We removed default fields at the past from jobs.
It worth that we will understand what are the defaults instead.
One can go to the main presubmits of kubevirt and see their config in case he need to remember what
they are for.
For example both `skip_report` and `optional` default is false, we can learn it from `pull-kubevirt-e2e-k8s-1.23-sig-network`, otherwise it won't consider it gating or won't show it on the PR.

Reply:
I think this topic doesn't have absolute truth. Its rather opinionated one. For me, in some cases defaults may be an overhead, in other cases I would like to see explicit definitions although they are trivial. If we want we can define a policy on how to write a proper Prow config and add it to the project doc, and add some sanity check on the syntax. But I don't see this discussion about the defaults as a blocker.

Reply:
> why skip_report true ?
> we do want to see the result on the PR, optional true is enough

[Fixed](https://github.com/kubevirt/project-infra/compare/83308dc3a07b358862c995faa9e6ec6201879276..5dae944164a57c5c72d9a42c9769cb7fb1b2cb25) thanks, I thought for some reason that this knob was related to the testgrid-dashboard.